server/models/Pet.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+const User = require('./User');
+const Appointment require('./Appointment');
+
+const petSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  birthday: {
+    type: Integer,
```

Comment:
Why are we defining Date as Integers?

Reply:
For date, I think we should use the format numeric format that you get while using Date.now() (1628868365654 - for example) and convert it to the XX/XX/XXXX format for display, the reason for that is that Graphql does not have a default date data type, and we would need to set a custom type for that (scalar type) which I am not familiar with yet, I want to ask Nelio/Fulton about it, but for the time being I put it as an integer. Hope this makes sense.

Reply:
Same response as before.